Source/AliveLib/MineCar.cpp

```diff
@@ -294,7 +294,7 @@ int CC MineCar::CreateFromSaveState_467740(const BYTE* pBuffer)
     pMineCar->field_118_tlvInfo = pState->field_4C_tlvInfo;
 
     pMineCar->field_11C_state = pState->field_50_state;
-    pMineCar->field_1BC = pState->field_52;
+    pMineCar->field_turnDirection_1BC = static_cast<MineCarDirs>(pState->field_52);
```

Comment:
can't the type of field_52 be  also changed to MineCarDirs and be renamed to represent what it actually is?